robot/impl/local_robot.go

```diff
@@ -501,12 +502,11 @@ func newWithResources(
 		return nil, err
 	}
 
-	modMgr, err := modmanager.NewManager(r)
+	modMgr, err := modmanager.NewManager(r, modmanageroptions.Options{UntrustedEnv: r.manager.opts.untrustedEnv})
```

Comment:
Why not directly just use cfg.UntrustedEnv here?

Reply:
We ended up following the pattern we have for weboptions to try and future proof this a bit. So if another option needs to be passed and added in the future we can add it without breaking the newManager interface.